packages/block-library/src/audio/index.js

```diff
@@ -13,12 +13,14 @@ import { createBlobURL } from '@wordpress/blob';
 
 export const name = 'core/audio';
 
+export const icon = <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M0,0h24v24H0V0z" fill="none" /><path d="m12 3l0.01 10.55c-0.59-0.34-1.27-0.55-2-0.55-2.22 0-4.01 1.79-4.01 4s1.79 4 4.01 4 3.99-1.79 3.99-4v-10h4v-4h-6zm-1.99 16c-1.1 0-2-0.9-2-2s0.9-2 2-2 2 0.9 2 2-0.9 2-2 2z" /></svg>;
```

Comment:
Out of curiosity, did you try leaving this in the attributes and importing like this:

```js
import { settings } from './index.js';

const {
  icon,
} = settings;
```

Alternately I think you could do something like `export const icon = settings.icon;`.

Definitely not 100% necessary but it feels a little cleaner to keep the icon in the settings object.

Reply:
Sounds like the best way to me. Plus the `settings` is already being exported

Reply:
@chrisvanpatten I was trying to follow the pattern that other blocks do for defining stuff outside of the settings object, e.g. the `supports` and `schema` objects in the Paragraph `index.js`.

Reply:
But that's a good point, @caxco93. In the Paragraph case, the things being defined outside of the `settings` object are not being exported. I think I will go ahead and change it according to @chrisvanpatten's suggestion.